### DISCUSSION TITLE
why you can break some of the new designed blocks (glowstone, stonecoal, torches on fence, goldore, ironore)

### DISCUSSION THREAD

**Discussion by MykeUhu** [TownyAdvanced/Towny#5243] [Q&A] [answered]
set up a server 1.17.1 
load towny.. 
most of the blocks you get the error (n .. wilderness.. or something) thats ok
but you can break glowstone, torches on fence, stonecoalore, the new ores (gold, iron)
can you fix it?

want start a little rp server ;)
and love towny ;)

PS: Sorry for my bad englisch, i just speak german

**Comment by LlmDl** [comment, marked as answer]
Use `/ta toggle wildernessuse` to fully enable use of the wilderness.

**Reply by MykeUhu** [reply]
I mean , the World is protected, but some Blocks you can break. In a Arena wich should be protected.

**Reply by LlmDl** [reply]
File a bug ticket.
